CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ release.
 - Fixed a bug in the <i>cnetwinnow</i> test that did not clean/remove it during test runs.
 - Fixed <i>findfeatures</i> instantiation and use of projection classes to correctly return geometry data from projected images and mosaics. [#4772](https://github.com/DOI-USGS/ISIS3/issues/4772)
 - Fixed `cubeit` attribute error to allow attribute specification on the output cube filename [#5234](https://github.com/DOI-USGS/ISIS3/issues/5234)
+- `campt` handles input band selection attribute correctly [#5234](https://github.com/DOI-USGS/ISIS3/issues/5235)
```

Comment:
Don't know why this sentence bothers me. Maybe instead "Fixed `campt` to handle input band selection attribute correctly"?